_data/docs.yml

```
@@ -46,6 +46,17 @@ apis:
       legacy: 1
       stable: 1
       nightly: 1
+  nx-cugraph:
+    name: nx-cugraph
+    path: cugraph # TODO: determine where to point nx-cugraph api docs
+    desc: 'nx-cugraph is a NetworkX backend that provides GPU acceleration to many popular NetworkX algorithms. With nx-cugraph, users can have GPU-based, large-scale performance without needing to change their pre-existing NetworkX code.'
+    ghlink: https://github.com/rapidsai/nx-cugraph
```

This is an internal repository link and won't work for public users. Just use the cuGraph link since nx-cugraph lives in the same repo. (This is how we handle dask-cudf above).
```suggestion
    ghlink: https://github.com/rapidsai/cugraph
```

---

I was under the impression that we would be migrating `nx-cugraph` out of cugraph as a part of the 24.10 release.

---

We can change this to use the cugraph repo for now and then just submit a followup PR when the nx-cugraph sources move to the new nx-cugraph repo later.